tests/e2e/pinned_releases.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+apt-get update && apt-get install -y apt-transport-https curl
```

Comment:
Can these apt-get commands be containerized so that this can run anywhere?
Or do we expect _this script_ to run in a container?

Reply:
The latter. My assumption was that prow would run this in a pod